### ISSUE TITLE
Wiring Diamgram is missed

### ISSUE THREAD

**Issue by ildarjaleh** [BCN3D/BCN3D-Moveo#6]
Please add wiring diagram to the instruction.

**Comment by AlexanderZhang11** [comment]
Yes, I am also puzzled by the wiring diagram problem. I really hope it can be updated as soon as possible.

**Comment by Mocsany** [comment]
Yes, please provide wiring diagram!

**Comment by knoxxr** [comment]
Need wiring diagram too.
I have ask about power supply.
Moveo use 24v power supply in BOM and use voltage converter 24v to 12v.
Why use 24v power instead 12v power.
Ramps 1.4 normaly use 12v power supply.
Is passible 12V, 320W power supply directly?

**Comment by xy23** [comment]
Hello I am close to finish the printing part of Moveo and I will start the wiring process in the next few Days. 
My Question is, is ther now any wiring Diagramm here on Github or can someon give me some hint where I can find one?

**Comment by fc-lapac** [comment]
Any news on that?

**Comment by michelgokan** [comment]
Where is the wiring diagram?

**Comment by vviudez** [comment]
No one has a wiring diagram?....
